src/services/safe_border.py

```diff
@@ -20,17 +20,29 @@ class WrongExitPeriod(Exception):
 
 
 class SafeBorder(Web3Converter):
+    """
+    The protocol can be in two states: Turbo and Bunker modes. Turbo mode is a usual state when requests are
+    finalized as fast as possible, while Bunker mode assumes a more prudent requests finalization and is activated
+    if it’s necessary to mitigate undesirable factors.
```

Comment:
No need to write what is bunker about
` and is activated
    if it’s necessary to mitigate undesirable factors`